dandiapi/zarr/migrations/0004_zarrarchive_embargoed_delete_embargoedzarrarchive.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.1.13 on 2024-08-21 16:06
+from __future__ import annotations
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('zarr', '0003_alter_embargoedzarrarchive_options_and_more'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='zarrarchive',
+            name='embargoed',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.DeleteModel(
+            name='EmbargoedZarrArchive',
+        ),
+    ]
```

Comment:
Don't we need to migrate the existing `EmbargoedZarrArchive` objects to the `ZarrArchive` table before deleting the `EmbargoedZarrArchive` table?

Reply:
There are no existing `EmbargoedZarrArchive`s, and aren't supported in any real way even if they did exist. So I think this achieves what we want.